requirements.txt

```diff
@@ -1,8 +1,8 @@
-django>=2.0,<2.2
+django>=2.0.6,<2.2
 swapper>=1.1.0,<1.2.0
 openwisp-utils>=0.2,<0.3
 djangorestframework>=3.8.2,<3.9
 passlib>=1.7.1,<1.8.0
 django-filter>=2.0.0,<2.1.0
 djangorestframework-link-header-pagination>=0.1.1,<0.2.0
-xhtml2pdf==0.2.2
+xhtml2pdf>=0.2.1,<0.2.2
```

Comment:
Hi. We are supposed to update the version to >=0.2.3, <0.3.0.

Reply:
In a moment!